File: advanced/Functions/Factorial/factorial.py
def factorial(num):

    # recursive:
    """if num == 0:
        return 1
    return factorial(num-1)*num"""

    # or normal-functionl:
    sum = 1
    for i in range(1, num+1):
        sum*= i

    return sum

print(factorial(10))

File: advanced/Functions/Factorial/factorial.js
function factorial(num) {
    // recursie:
    if (num == 0){ return 1}
    return factorial(num-1)* num

    // normal functional
    /*let factorial =1;
    for (let i=1; i< num+1; i++){
        factorial*= i
    }
    return factorial*/
}

console.log(factorial(10))